From d335fa8196c6ca92e3975a5ae950de08e9578f9b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <mika.bostrom@nomovok.com>
Date: Thu, 19 Apr 2012 17:55:47 +0300
Subject: [PATCH 1/6] Default to package build

OBS builds appear to break when CONFIG additions are placed
in qmake invocation.
---
 qt3d.pro |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/qt3d.pro b/qt3d.pro
index ba83747..a731786 100644
--- a/qt3d.pro
+++ b/qt3d.pro
@@ -1,5 +1,9 @@
 TEMPLATE = subdirs
 
+# Force on for modular (packaging) builds
+# [when building in OBS, 'qmake CONFIG+=package' fails]
+CONFIG += package
+
 module_qt3d_src.subdir = src
 module_qt3d_src.target = module-qt3d-src
 
-- 
1.7.5.4

